Gemfile

```diff
@@ -15,6 +15,19 @@ gem "twitter-bootstrap-rails"
 
 gem 'devise'
 gem 'rubocop'
+gem 'rb-readline'
+
+group :development, :test do
+  gem 'rspec-rails', '~> 3.0'
```

Comment:
Don't change it now. But take into account that you adding rspec-rails, for example, to the development group and thats not neccesaary